.eslintrc

```diff
@@ -62,6 +62,7 @@
     "no-mixed-requires": 0,                                       // http://eslint.org/docs/rules/no-mixed-requires
     "no-multi-str": 0,                                            // http://eslint.org/docs/rules/no-multi-str
     "no-nested-ternary": 0,                                       // http://eslint.org/docs/rules/no-nested-ternary
+    "no-param-reassign": 0,                                       // https://eslint.org/docs/rules/no-param-reassign
```

Comment:
I'd like that `setConfig` return a new config object so we maintain immutability. Then in `config/index.js` we can do `const config = require(configFile).setConfig(baseConfig);` and export config. What do you think?

Reply:
I think that this rule (off) could be useful in several places. Maybe we should discuss it.